autoload/vaffle/file.vim

```diff
@@ -2,13 +2,14 @@ let s:save_cpo = &cpoptions
 set cpoptions&vim
 
 
-function! vaffle#file#open(items) abort
+function! vaffle#file#open(items, ...) abort
   if len(a:items) == 1
     let path = a:items[0].path
     if isdirectory(path)
       call vaffle#init(path)
```

Comment:
If user press `t` on the directory, Vaffle open it in the same buffer. Maybe users expect to open the directory in new tab.

Reply:
I faced the same problem... 😢 
Sorry, I will check it.

Reply:
I think `open_cmd` (can be any string) argument makes difficult to handle each cases.

How about using something like a mode instead of a command string?
e.g. `open_mode` (`''` | `'tab'` | ...)

Reply:
It's great 👍 
I apply the changes

Reply:
I just introduced `open_mode` into `vaffle#file#open`.
Is is enough ? or it is better to put `open_mode` into the other functions such as `vaffle#init` ?

Reply:
Thanks! I think it's enough now.